.github/workflows/node.js.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 19.x]

```

Comment:
is there a good reason to support multiple node versions?  my suggestion would be to stick to one version for now and get everything working, and then if we want to we can revisit this and add more versions.

Reply:
In the past it was a practice of the old repo maintainer. He tested on old versions in Team City. Until someone come and tell him to stop supporting legacy. The argument is that many users use legacy libraries and run on a legacy node binary. If we stop testing we don't guarantee support for them.

18.x is Active LTS and will be dropped in 2025-04-30

https://en.wikipedia.org/wiki/Node.js#Releases